bclientapi/bclientapi.go

```diff
@@ -151,7 +155,38 @@ func (ia *itemAttributes) SendNewTransactionRequest() error {
 
 	buf, _ := json.Marshal(cmdlist)
 
-	transErr := ia.createFileTransAction(buf)
+	transaction, transErr := ia.createFileTransAction(buf)
 
-	return transErr
+	return transaction, transErr
+}
```

Comment:
FYI, in this situation (assigning into variables that are then returned as-is) you can also do

    return ia.createFileTransAction(buf)